### PR TITLE
Add feature remove role by removing reaction and fix the bug when kic…

### PR DESCRIPTION
**Pull request by dongtandung2001**
…k the bot that makes bot crash due to None message object

### CODE REVIEW

File: src/util/MusicSingleton.js [hunk 1]
```diff
@@ -125,9 +125,12 @@ class MusicSingleton {
...
 
   stop(message) {
-    if (!message.member.voice.channel) {
-      return message.reply('Please join voice channel first!');
+    if (message) {
+      if (message.member.voice.channel) {
+        return message.reply('Please join voice channel first!');
+      }
     }
+
```
Comment (evanugarte) [inline src/util/MusicSingleton.js:133]: do we need this?

**Review by pnavab — APPROVED** [review]
tested and it works

File: src/util/MusicSingleton.js [hunk 2]
```diff
@@ -125,9 +125,11 @@ class MusicSingleton {
   }
 
   stop(message) {
-    if (!message.member.voice.channel) {
+    if (message && message.member.voice.channel) {
```
Comment (evanugarte) [inline src/util/MusicSingleton.js:128]:
```suggestion
    if (!message || !message.member.voice.channel) {
```

File: src/util/MusicSingleton.js [hunk 3]
```diff
@@ -125,9 +125,11 @@ class MusicSingleton {
...
       return message.reply('Please join voice channel first!');
     }
```
Comment (evanugarte) [inline src/util/MusicSingleton.js:130]: assuming this passes lint
```suggestion
    if (!message) return;
    if (!message.member.voice.channel) {
      return message.reply('Please join voice channel first!');
    }
```

**Pull request merged**